core/bones/json.py

```diff
@@ -1,21 +1,36 @@
 import ast
 import json
-from typing import Union
+import jsonschema
+from typing import Union, Mapping
 from viur.core.bones.base import ReadFromClientError, ReadFromClientErrorSeverity
 from viur.core.bones.raw import RawBone
 
 
 class JsonBone(RawBone):
     """
     This bone saves its content as a JSON-string, but unpacks its content to a dict or list when used.
+    :param schema If provided we can controll what data we accept.
+    Example:
+        >>>schema = {"type" : "object",
+        >>>     "properties" : {
+        >>>"price" : {"type" : "number"},
+        >>>"name" : {"type" : "string"},
+        >>>}}
+        We only accept the json if the price is a number and name is a string.
```

Comment:
```suggestion
        >>> schema = {"type" : "object",
        >>>     "properties" : {
        >>>         "price" : {"type" : "number"},
        >>>         "name" : {"type" : "string"},
        >>>     }
        >>> }
        This will only accept the provided JSON when price is a number and name is a string.
```